frontend/packages/console-app/src/components/modals/clone/clone-pvc-modal.tsx

```diff
@@ -47,12 +47,14 @@ import {
 } from '@console/internal/module/k8s';
 import { RedExclamationCircleIcon, isCephProvisioner } from '@console/shared';
 import { getRequestedPVCSize } from '@console/shared/src/selectors';
+import { StorageClassDropdown } from '../../../../../../public/components/utils/storage-class-dropdown';
 import { getPVCAccessModes, AccessModeSelector } from '../../access-modes/access-mode';
 
 import './_clone-pvc-modal.scss';
 
 const ClonePVCModal = withHandlePromise((props: ClonePVCModalProps) => {
   const { t } = useTranslation();
+  const [storageClass, setStorageClass] = React.useState('');
```

Comment:
```suggestion
  const [storageClassName, setStorageClassName] = React.useState(resource.spec?.storageClassName);
```
Provide an older storage class name as a default option for the users who don't want the cross-storageClass selection.